ecommerce/core/client.py

```diff
@@ -145,12 +240,94 @@ def get_highest_sort_order_for_cart_discount_without_codes(self) -> Dict:
             "GET",
             "cart-discounts",
             params={
-                "where": "requiresDiscountCode=false and target(type=\"lineItems\")",
+                "where": 'requiresDiscountCode=false and target(type="lineItems")',
                 "sort": ["sortOrder desc"],
                 "limit": 1,
             },
         )
 
+    def create_cart_discount(
+        self,
+        *,
+        key,
+        name,
+        description,
+        value,
+        cartPredicate,
+        sortOrder,
+        customFields,
+    ) -> Optional[Dict]:
+        """
+        Create a new cart discount.
+
+        Args:
+            payload (dict): Payload containing cart discount data.
+
+        Returns:
+            Dict: Created cart discount data or None if request fails.
+        """
+        payload = {
+            "key": key,
+            "name": {"en-us": name},
+            "description": {"en-us": description},
+            "value": value,
+            "cartPredicate": cartPredicate,
+            "target": {
+                "type": "totalPrice",
+            },
```

Comment:
Pass it like this instead so that lineItem discounts can also be created using the same util
```suggestion
            "target": target,
```